pkg/cmd/tknpac/webhook/github.go

```diff
@@ -0,0 +1,161 @@
+package webhook
```

Comment:
maybe we should move that class to pkg/cli/info insted since it's cli specific

Reply:
well I wrote it as a separate command
may be in future we want to allow having a cmd to just configure webhook
lets say after creating repo .. the webhook creation fails due to some reason
how would user can just do webhook

Reply:
yeah we have a flag

Reply:
i will move to info and later we can decide

Reply:
yeah like bootstrap github-app we should have "bootstrap webhook"

Reply:
https://github.com/openshift-pipelines/pipelines-as-code/issues/602